setup.py

```diff
@@ -1,8 +1,14 @@
 from setuptools import setup, find_packages
 
+version = "Unknown"
+for line in open("metasv/_version.py"):
+    if line.startswith("__version__"):
+        version = line.strip().split("=")[1].strip()
```

Comment:
It's upto us to parse the version from the version file. You shouldn't be
importing from the metasv package in setup.py.
http://stackoverflow.com/questions/458550/standard-way-to-embed-version-into-python-package

On Thu, Jan 29, 2015 at 6:03 PM, John Mu <notifications@github.com> wrote:

> In setup.py
> <https://github.com/bioinform/metasv/pull/21#discussion_r23819832>:
>
> > @@ -1,8 +1,14 @@
> >  from setuptools import setup, find_packages
> >
> > +version = "Unknown"
> > +for line in open("metasv/_version.py"):
> > +    if line.startswith("__version__"):
> > +        version = line.strip().split("=")[1].strip()
>
> lol, is this typical for python? seems kind of odd
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bioinform/metasv/pull/21/files#r23819832>.
>



-- 

<http://www.bina.com>

*Marghoob Mohiyuddin, PhD  *//  Sr. Bioinformatics Scientist, Bina
Technologies
E. marghoob@bina.com

*Learn More*  //  video <http://www.bina.com/product.html>  //  news
<http://www.bina.com/press-coverage.html>  //  posters
<http://f1000.com/posters/search?query=bina+technologies>  //  science
<http://www.bina.com/science.html>  //  platform
<http://www.bina.com/platform.html>
*Test Drive Bina  //  Experience NextGen analysis for yourself *
<http://www.bina.com/test-drive.html>